Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -666,6 +651,51 @@ namespace OpenInfraPlatform {
 					return;
 				}
 
+				/**********************************************************************************************/
+				/*! \brief Calculates the angle on the circle of the trimming point.
+				* \param[in] circle				A pointer to data from c\ IfcCircle.
+				* \param[in] trimmingVector		The trimming of the curve as saved in IFC model
+				* \param[in] circle_center		Coordinates of the center of the circle.
+				* \param[in] circle_radius		Radius of the circle. 
+				* \return						Angle on the circle of the trimming point.
+				*/
+				double  calculateTrimmingPointOnCircle(const EXPRESSReference<typename IfcEntityTypesT::IfcCircle>& circle,
+					const std::vector<std::shared_ptr<typename IfcEntityTypesT::IfcTrimmingSelect>>& trimmingVector,
+					carve::geom::vector<3> circle_center,
+					double circle_radius) const throw(...)

```

Comment:
if the values are not changed in the function, use `const`

Reply:
Resolved:  4878063